modules/ppcp-settings/resources/js/Components/Screens/Settings/Components/Settings/Blocks/SavePaymentMethods.js

```diff
@@ -5,7 +5,7 @@ import { ControlToggleButton } from '../../../../../ReusableComponents/Controls'
 import { SettingsHooks } from '../../../../../../data';
 import { useMerchantInfo } from '../../../../../../data/common/hooks';
 
-const SavePaymentMethods = () => {
+const SavePaymentMethods = ( { ownBradOnly } ) => {
```

Comment:
```suggestion
const SavePaymentMethods = ( { ownBrandOnly } ) => {
```
🙈 

Super minor :) but there's `ownBradOnly` typo in a few places in this PR.